arithmetization/src/main/java/net/consensys/linea/zktracer/container/stacked/ModuleOperationStackedSet.java

```diff
@@ -41,6 +41,7 @@ public class ModuleOperationStackedSet<E extends ModuleOperation> extends Stacke
   private static final Logger log = LoggerFactory.getLogger(ModuleOperationStackedSet.class);
   private final CountOnlyOperation lineCounter = new CountOnlyOperation();
   @Getter private boolean conflationFinished = false;
+  private boolean duplicateAlreadyRemoved = false;
```

Comment:
I added this because I think we don't call `lineCount` when the conflation is already created. So we need to know if we already delete duplicates.